docs/integrate/debezium/tutorial.md

```
@@ -0,0 +1,277 @@
+## Configure a source connector
+
+Let’s create a `connector.json` file as follows:
+
+```json
+{
+    "name": "mssql-source-tbltest",
+    "config": {
+        "connector.class": "io.debezium.connector.sqlserver.SqlServerConnector",
+        "tasks.max": "1",
+        
+		"database.history.kafka.bootstrap.servers": "host.docker.internal:9092",
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)